README.md

```diff
@@ -2,12 +2,6 @@
 
 # PeerScout API
 
-To test the access token
-
-```bash
-make peerscout-api-start
```

Comment:
perhaps still keep the comment regarding starting the api?

Reply:
Actually there was two and I removed one of them so there still command for starting the api

Reply:
Ah okay, sorry, I didn't see that.